src/policy/vmspace.rs

```diff
@@ -224,6 +224,14 @@ impl<VM: VMBinding> VMSpace<VM> {
             start: start.align_down(BYTES_IN_PAGE),
             end: end.align_up(BYTES_IN_PAGE),
         });
+
+        if self.common.needs_log_bit {
+            if let MetadataSpec::OnSide(side) = *VM::VMObjectModel::GLOBAL_LOG_BIT_SPEC {
+                side.bset_metadata(start, size);
+            } else {
+                unimplemented!("We cannot bulk set unlogged bit.")
```

Comment:
If a VM uses in-header unlog bits, the VM can still set the unlog bit to 1 in the boot image and make it behave normally.  It is not necessary to panic.

Reply:
Ah that's fair

Reply:
Done